index.html

```diff
@@ -31,13 +32,210 @@ <h2 id="job-title">I am a Software Developer</h2>
                     <p id="introduction">I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesistate to contact me</p>
                 </div>
                 <ul class="social">
-                    <li><a href=""><img class="icon-interactions" src="images/github.png" alt="github-image"></li></a>
-                    <li><a href=""><img class="icon-interactions" src="images/Group.png" alt="union-icon"></li></a>
-                    <li><a href=""><img class="icon-interactions" src="images/hand.png" alt="hand"></li></a>
-                    <li><a href=""><img class="icon-interactions" src="images/twitter.png" alt="twitter bird"></li></a>
-                    <li><a href=""><img class="icon-interactions" src="images/Vector.png" alt="M-sign"></li></a>
+                    <li><a href="#"><img class="icon-interactions" src="images/github.png" alt="github-image"></li></a>
+                    <li><a href="#"><img class="icon-interactions" src="images/linkedin.png" alt="union-icon"></li></a>
+                    <li><a href="#"><img class="icon-interactions" src="images/hand.png" alt="hand"></li></a>
+                    <li><a href="#"><img class="icon-interactions" src="images/twitter.png" alt="twitter bird"></li></a>
+                    <li><a href="#"><img class="icon-interactions" src="images/Vector.png" alt="M-sign"></li></a>
                 </ul> 
             </section>
+            <section class="work">
+                <div>
+                    <div class="recent-work">
+                        <h2 id="work-intro">My Recent Work</h2>
+                        <a id="yoga-image" href=""><img src="images/Img Placeholder-1.png" alt="yoga girl"></a>
+                    </div>
+                    <div class="multi-post-div">
+                        <h2 id="multi-post-title" >Multi-Post Stories</h2>
+                        <div id="work-description">
+                            <p id="work-font">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                        </div>
+                        <div class="button-div">
+                            <ul class="button-list">
+                                <li><a href="#"><button id="multi-post-button">css</button></a></li>
+                                <li><a href="#"><button id="multi-post-button">html</button></a></li>
+                                <li><a href="#"><button id="multi-post-button">bootstrap</button></a></li>
+                                <li><a href="#"><button id="multi-post-button">Ruby</button></a></li>
```

Comment:
Kindly, in the past revision I told you about the IDs that must be unique on your page, there should be no equal IDs, I recommend you use "classes" to use it with CSS. You must apply this in all your ID on your page